packages/sales-api-service/src/services/__tests__/permissions.service.spec.js

```diff
@@ -98,41 +88,27 @@ describe('permissions service', () => {
           licensee: {
             firstName: 'Fester',
             lastName: 'Tester',
-            birthDate: moment(now)
-              .subtract(SENIOR_MIN_AGE, 'years')
-              .format('YYYY-MM-DD')
+            birthDate: moment(now).subtract(SENIOR_MIN_AGE, 'years').format('YYYY-MM-DD')
           }
         },
         'Web Sales'
       )
-      const block1 = moment(now)
-        .add(1, 'hour')
-        .startOf('hour')
-        .add(1, 'day')
-        .format('HHDDMMYY')
+      const block1 = moment(now).add(1, 'hour').startOf('hour').add(1, 'day').format('HHDDMMYY')
       const expected = new RegExp(`^${block1}-2WC1SFT-[A-Z0-9]{5}[0-9]$`)
       expect(number).toMatch(expected)
     })
   })
 
   describe('calculateEndDate', () => {
-    it('calculates 365 days for 1 year licences outside of a leap year', async () => {
+    it('calculates 364 days for 1 year licences outside of a leap year', async () => {
       const startDate = moment('2019-01-01')
       const endDate = await calculateEndDate({ permitId: 'e11b34a0-0c66-e611-80dc-c4346bad0190', startDate: startDate })
-      expect(endDate).toEqual(
-        moment(startDate)
-          .add(365, 'days')
-          .toISOString()
-      )
+      expect(endDate).toEqual(moment(startDate).add(364, 'days').endOf('day').toISOString())
```

Comment:
I know you're only modifying what was already here, but given you've got a fixed start date, I'd go for a hard-coded end date rather than calculating it with moment. I think it would be slightly clearer.

Reply:
That sounds a lot clearer to me too. Fixed!